packages/core/src/modules/sd-jwt-vc/SdJwtVcService.ts

```diff
@@ -85,16 +88,26 @@ export class SdJwtVcService {
       kid: issuer.kid,
     } as const
 
-    const sdjwt = new SDJwtInstance({
-      hasher: this.hasher,
+    const sdjwt = new SDJwtVcInstance({
+      ...this.getBaseSdJwtConfig(agentContext),
       signer: this.signer(agentContext, issuer.key),
       hashAlg: 'sha-256',
```

Comment:
Hmm hash-alg could be different right in the verify/present method? We only support sha-256 at the moment, but in the future it could be different and we don't want to set that default

Reply:
so only added the salt generator

Reply:
My point was more that the hasher method has a default but the hashAlg not, they should always be grouped together. This might be the wrong place/PR to fix that though. Maybe the `public get hasher()` should return the alg as well?